modules/ui/sections/background_list.js

```diff
@@ -189,6 +192,36 @@ export function uiSectionBackgroundList(context) {
             .classed('imagery-show-icons', showIcons)
             .html(function(d) { return d.label(); });
 
+        enter
+            .append('button')
+            .attr('class', 'stamp background-favorite-button')
+            .classed('active', d => categoryMapping(d).id === 'favs')
+            .on('click', (_, d) => {
+                var group = categoryMapping(d);
+                if (group.id === 'favs') {
+                    delete _favouriteLayerIdMappings[d.id];
+                } else {
+                    _favouriteLayerIdMappings[d.id] = categoryGroups.favs;
+                }
+                prefs('background-favorites', JSON.stringify(Object.keys(_favouriteLayerIdMappings)));
+                _sources = updateSources();
+                reRender();
+                d3_selectAll('.layer-background-list li').filter(item => item === d)
+                    .transition()
+                    .duration(300)
+                    .ease(d3_easeCubicInOut)
+                    .style('background-color', '#ccc')
+                    .transition()
+                    .duration(300)
+                    .ease(d3_easeCubicInOut)
+                    .style('background-color', null);
+            })
+            .call(svgIcon('#maki-heart-11'))
+            .call(uiTooltip()
+                .title(t.html('background.add_remove_favorite'))
```

Comment:
These “tooltips” show up immediately, covering up the layer name. Sometimes I also see the tooltip misaligned so that it points to the strip of buttons along the right edge. I think we could just use normal HTML tooltips for the ❤️ 🌐 ⭐ icons, similar to the ↩️ ℹ️ 🚮 buttons in preset fields.